src/main/java/net/sf/jabref/gui/errorconsole/ErrorConsoleView.java

```diff
@@ -0,0 +1,141 @@
+        errorConsole.setDialogPane(pane);
+        errorConsole.setResizable(true);
+        errorConsole.show();
+    }
+
+    @FXML
+    private void initialize() {
+        ButtonBar.setButtonData(copyLogButton, ButtonBar.ButtonData.LEFT);
+        ButtonBar.setButtonData(createIssueButton, ButtonBar.ButtonData.LEFT);
+
+        ObservableList<ObservableMessageWithPriority> masterData = ObservableMessages.INSTANCE.messagesPropety();
+        listViewStyle();
+        allMessage.setItems(masterData);
+
+        //set console icon instead the error icon and insert a description
+        Text graphic = new Text(IconTheme.JabRefIcon.CONSOLE.getCode());
+        graphic.getStyleClass().add("icon");
+        String headerLabel = Localization.lang("We now give you an insight into the inner workings of JabRef's brain.") + " "
+                + Localization.lang("This might help to diagonalize the root of problems.") + " " + System.lineSeparator()
+                + Localization.lang("Please use the button below to inform the developers about an issue.");
+        descriptionLabel.setText(headerLabel);
+        descriptionLabel.setGraphic(graphic);
+    }
+
+    @FXML
+    private void copyLogButton() {
+        errorViewModel.copyLog();
+    }
+
+    @FXML
+    private void createIssueButton() {
+        errorViewModel.reportIssue();
+    }
+
+    @FXML
+    private void closeErrorDialog() {
+        Stage stage = (Stage) closeButton.getScene().getWindow();
+        stage.close();
+    }
+
+    //style the list view with icon and message color
+    private void listViewStyle() {
+        // handler for listCell appearance (example for exception Cell)
+        allMessage.setCellFactory(
+                new Callback<ListView<ObservableMessageWithPriority>, ListCell<ObservableMessageWithPriority>>() {
+                    @Override
+                    public ListCell<ObservableMessageWithPriority> call(
+                            ListView<ObservableMessageWithPriority> listView) {
+                        return new ListCell<ObservableMessageWithPriority>() {
+
+                            @Override
+                            public void updateItem(ObservableMessageWithPriority omp, boolean empty) {
+                                super.updateItem(omp, empty);
+                                if (omp != null) {
+                                    setText(omp.getMessage());
+                                    getStyleClass().clear();
+                                    Text graphic = new Text();
+                                    graphic.getStyleClass().add("icon");
+                                    if (omp.getPriority() == MessagePriority.HIGH) {
+                                        getStyleClass().add("exception");
+                                        graphic.setText(IconTheme.JabRefIcon.INTEGRITY_FAIL.getCode());
+                                    } else if (omp.getPriority() == MessagePriority.MEDIUM) {
```

Comment:
Refactor, by moving some of the code to ObservableMessageWithPriority.
It should be as simple as `getStyleClass.add(omp.getStyleClass())` and `setGraphic(omp.getIcon())`.